.repo-metadata.json

```diff
@@ -1,8 +1,9 @@
 {
-    "name": "repo-automation-bots",
-    "name_pretty": "Repo Automation Bots",
-    "release_level": "beta",
-    "language": "nodejs",
-    "repo": "googleapis/repo-automation-bots",
-    "distribution_name": "repo-automation-bots"
-  }
+  "name": "repo-automation-bots",
+  "name_pretty": "Repo Automation Bots",
+  "release_level": "beta",
+  "language": "nodejs",
+  "repo": "googleapis/repo-automation-bots",
+  "distribution_name": "repo-automation-bots",
+  "codeowner_team": "@googleapis/github-automation"
```

Comment:
What does this do?

Reply:
There is a PR in synthtool to use this field to populate the CODEOWNERS file.  We need this to ensure new repositories get a CODEOWNERS, as most of our new repos are missing it.